### PR TITLE
Objectives polish

### PR DESCRIPTION
**Pull request by obrakmann**

### CODE REVIEW

File: OpenRA.Mods.Common/Traits/Player/ConquestVictoryConditions.cs [hunk 1]
```diff
@@ -16,7 +16,10 @@ namespace OpenRA.Mods.Common.Traits
 	public class ConquestVictoryConditionsInfo : ITraitInfo, Requires<MissionObjectivesInfo>
 	{
 		[Desc("Delay for the end game notification in milliseconds.")]
-		public int NotificationDelay = 1500;
+		public readonly int NotificationDelay = 1500;
+
+		[Desc("Description of the objective.")]
+		public readonly string Objective = "Destroy all opposition!";
```
Comment (Mailaender) [inline OpenRA.Mods.Common/Traits/Player/ConquestVictoryConditions.cs]: Could use a `[Translate]` attribute.

File: OpenRA.Mods.Common/Traits/Player/StrategicVictoryConditions.cs [hunk 2]
```diff
@@ -31,7 +31,10 @@ public class StrategicVictoryConditionsInfo : ITraitInfo, Requires<MissionObject
 		public readonly float RatioRequired = 0.5f;
 
 		[Desc("Delay for the end game notification in milliseconds.")]
-		public int NotificationDelay = 1500;
+		public readonly int NotificationDelay = 1500;
+
+		[Desc("Description of the objective")]
+		public readonly string Objective = "Hold all the strategic positions!";
```
Comment (Mailaender) [inline OpenRA.Mods.Common/Traits/Player/StrategicVictoryConditions.cs]: Same here: missing `[Translate]`

**Comment by obrakmann** [comment]
Isn't the translation project pretty much dead? Anyway, updated.

**Comment by Mailaender** [comment]
It has a low priority, but now I am not the only non-native English speaking developer anymore so a pull request might have a chance to get merged.

**Comment by Mailaender** [comment]
:+1: That also annoyed me. Also yay for less hard-coded strings.

**Comment by abcdefg30** [comment]
Looks also good to me. :+1: / :white_check_mark:

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/c326a9ce388f74de5d20f973708e02443d7945e3...f3745ee5c30f8c0f8e4c50047798fb9d52f05cd0)

**Pull request merged**